app/assets/scss/application.scss

```diff
@@ -23,30 +23,21 @@ $path: "/static/images/";
 
 // Digital Marketplace Front-end toolkit styles
 @import "toolkit/_breadcrumb.scss";
-@import "toolkit/_browse-list.scss";
 @import "toolkit/_buttons.scss";
-@import "toolkit/_contact-details.scss";
-@import "toolkit/_document.scss";
```

Comment:
We are using this for the brief-responses.html template. This pulls in the toolkit documents template (https://github.com/alphagov/digitalmarketplace-frontend-toolkit/blob/master/toolkit/templates/documents.html) which requires the scss file for it.

On another note, github search sucks. If you search for `document` in the briefs repo you get nothing. If you search `documents` then you do find it...

Reply:
👍  I put it back